cf_units/__init__.py

```diff
@@ -1849,10 +1849,10 @@ def __ne__(self, other):
         """
         return not self == other
 
-    def convert(self, value, other, ctype=FLOAT64):
+    def convert(self, value, other, ctype=FLOAT64, inplace=True):
```

Comment:
I'm not convinced this is a sensible default. I suspect the majority of users will want to *not* modify the data they are converting in-place.